src/dashboard/get_from_parquet/get_from_parquet.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import awswrangler
+import pyarrow
+
+from shared import decorators, functions
+
+
+@decorators.generic_error_handler(msg="Error retrieving data from parquet")
+def from_parquet_handler(event, context):
+    """manages event and returns parquet file as json dict"""
+    del context
+    s3_bucket_name = os.environ.get("BUCKET_NAME")
+    try:
+        s3_path = event["queryStringParameters"]["s3_path"]
+        df = awswrangler.s3.read_parquet(f"s3://{s3_bucket_name}/{s3_path}")
+    except (KeyError, FileNotFoundError, pyarrow.lib.ArrowInvalid):
+        res = functions.http_response(404, "S3_path not found")
+        return res
+    if output_type := event["queryStringParameters"].get("type"):
```

Comment:
This path doesn’t allow for passing type=json

Reply:
i went back and forth on this - i'll add an explicit case